packages/typespec-ts/test/modularIntegration/versioningAdded.spec.ts

```diff
@@ -5,7 +5,8 @@ describe("VersioningAdded Rest Client", () => {
   let client: AddedClient;
 
   beforeEach(() => {
-    client = new AddedClient("http://localhost:3002", "v2", {
+    client = new AddedClient("http://localhost:3002", {
+      version: "v2",
```

Comment:
Since this is optional, could you revomve this setting? I believe the default value logic would cover this. And also apply to other versioning spec ts files and the same as azure modular and regular modular cases.